skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/LinuxSoftwareRedrawer.kt

```diff
@@ -0,0 +1,70 @@
+package org.jetbrains.skiko.redrawer
+
+import org.jetbrains.skia.Surface
+import kotlinx.coroutines.*
+import kotlinx.coroutines.swing.Swing
+import org.jetbrains.skiko.*
+
+internal class LinuxSoftwareRedrawer(
```

Comment:
LinuxSoftwareRedrawer looks almost the same as WindowSoftwareRedrawer (same for LinuxSoftwareContextHandler)

I think, it will help us in the future, if we create something like AbstractSoftwareRedrawer and AbstractSoftwareContextHandler.

Reply:
Done.